mbtiles/src/summary.rs

```diff
@@ -222,8 +222,10 @@ mod tests {
 
     #[actix_rt::test]
     async fn summary() -> MbtResult<()> {
-        let mbt = Mbtiles::new("../tests/fixtures/mbtiles/world_cities.mbtiles")?;
+        let mbt = Mbtiles::new("file:world_cities?mode=memory&cache=shared")?;
```

Comment:
You don't need a reusable (shared) connection here i don't think.  You would only need it when you create the (in-memory) file with `...?mode=memory&cache=shared`, and later pass the same identical string to some API to work on that "file".

```suggestion
        let mbt = Mbtiles::new(":memory:")?;
```